content/applications/inventory_and_mrp/manufacturing/subcontracting/basic_subcontracting_lead_times.rst

```diff
@@ -0,0 +1,101 @@
+   should not be viewed as guarantees.
+
+Configuration
+=============
+
+When using the :doc:`basic subcontracting <subcontracting_basic>` workflow to manufacture a
+product, a company is not responsible for supplying the subcontractor with the necessary components.
+This means that the only factors affecting the delivery date of a product are the amount of time it
+takes the subcontractor to manufacture and delivery it.
+
+By assigning a product's subcontractor a *delivery lead time* that considers both of these factors,
+the *Expected Arrival* date displayed on purchase orders (POs) for the product more accurately
+reflects the amount of time required for both manufacturing and delivery.
+
+Product delivery lead time
+--------------------------
+
+To set a *delivery lead time* for a product's subcontractor, navigate to :menuselection:`Inventory
+app --> Products --> Products`, and select a subcontracted product.
+
+Then, select the :guilabel:`Purchase` tab on the product's page. If the subcontractor has not
+already been added as a vendor, do so now by clicking :guilabel:`Add a line`, and selecting the
+subcontractor in the :guilabel:`Vendor` column.
+
+Once the subcontractor has been added, enter the number of days it takes them to manufacture and
+deliver the product, in the :guilabel:`Delivery Lead Time` column.
+
+.. image:: basic_subcontracting_lead_times/delivery-lead-time.png
+   :align: center
+   :alt: The Delivery Lead Time field for a subcontractor, on the Purchase tab of a product page.
+
+.. note::
+   Multiple subcontractors can be added to the :guilabel:`Purchase` tab on a product's page, and a
+   different :guilabel:`Delivery Lead Time` can be set for each.
+
+Lead time workflow
+==================
+
+After setting a *delivery lead time* for a product's vendor, create an |RfQ| by navigating to
+:menuselection:`Purchase app --> Orders --> Purchase Orders`, and clicking :guilabel:`New`.
+
+Specify the subcontractor in the :guilabel:`Vendor` field. Then, add the product in the
+:guilabel:`Products` tab by clicking :guilabel:`Add a product`, selecting the product in the
+:guilabel:`Product` column, and adding a quantity in the :guilabel:`Quantity` column.
+
+Once a product has been added, the :guilabel:`Expected Arrival` field on the |RfQ| auto-populates
+with a date that reflects the vendor's *delivery lead time*, as specified on the product's page.
+
+If the date needs to be adjusted, click on the :guilabel:`Expected Arrival` field to open a calendar
+pop-up, and select the desired date. Make sure not to choose a date sooner than the one that was
+auto-populated, unless the subcontractor has confirmed that they are able to deliver the product by
+that date.
+
+Finally, click :guilabel:`Confirm Order` on the |RfQ| to turn it into a |PO|. At this point, the
+subcontractor should begin manufacturing the subcontracted product, before delivering it by the
+*Expected Arrival* date.
```

Comment:
```suggestion
:guilabel:`Expected Arrival` date.
```
suggesting the application of a GUI here because, technically, the reader will still be on the PO form, which shows that field on the UI